user_test.go

```diff
@@ -44,3 +44,30 @@ func TestGetUser(t *testing.T) {
 		t.Fatalf("expected %s, got %s\n", primaryUser, user.Email)
 	}
 }
+
+func TestCreateUser(t *testing.T) {
+	var (
+		apiKey      = os.Getenv("ZOOM_API_KEY")
+		apiSecret   = os.Getenv("ZOOM_API_SECRET")
+		primaryUser = os.Getenv("ZOOM_EXAMPLE_EMAIL")
+	)
+
+	APIKey = apiKey
+	APISecret = apiSecret
+
+	opts := CreateUserOptions{
+		Action: Create,
+		UserInfo: CreateUserInfo{
+			Type:  Basic,
+			Email: primaryUser,
+		},
+	}
+	user, err := CreateUser(opts)
```

Comment:
What happens if a user already exists with this email? Maybe it would be a good idea to create a somewhat more randomized user and then also delete that user as part of this test. This would obviously require adding the delete endpoint. These integration tests are operating on a real account, and I'd like to see them have a net-neutral effect.